CHANGELOG.md

```diff
@@ -264,6 +860,7 @@ This simply ensures we did not leave any unattended bugs before going for The Lo
 - Track channel state to allow removing channels user is not in ([#2058](https://github.com/thelounge/thelounge/pull/2058) by [@xPaw](https://github.com/xPaw))
 - Allow scaling the page ([#1910](https://github.com/thelounge/thelounge/pull/1910) by [@xPaw](https://github.com/xPaw))
 - Fix `bind` not being passed to `irc-framework` ([#2071](https://github.com/thelounge/thelounge/pull/2071) by [@xPaw](https://github.com/xPaw))
+- Set yarn cache folder in the packages folder ([#2979](https://github.com/thelounge/thelounge/pull/2979) by [@xPaw](https://github.com/xPaw))
```

Comment:
@astorije You put this in the wrong release section.

Reply:
🤦‍♂️ Fixed.